test/models/concerns/host_managed_extensions_test.rb

```diff
@@ -79,18 +77,18 @@ def test_update_does_not_update_content_host
     end
 
     def test_update_with_cv_env
-      host = FactoryGirl.create(:host, :with_content, :content_view => @library_view, :lifecycle_environment => @library)
-      host.content_view = @library_view
-      host.lifecycle_environment = @library
-      assert host.save!
+      host = FactoryGirl.create(:host, :with_content, :content_view => view, :lifecycle_environment =>  library)
```

Comment:
I think this is the reason it failed, library isn't defined here, and neither is view.